Source/VSTeam.psm1

```diff
@@ -21,4 +21,11 @@ if ($null -ne $env:TEAM_PROJECT) {
    if (Get-VSTeamProject | Where-Object ProjectName -eq $env:TEAM_PROJECT) {
       Set-VSTeamDefaultProject -Project $env:TEAM_PROJECT
    }
+}
+
+# Check to see if there is a newwer version of the module
+$latestMod = Find-Module -Name VSTeam
+
+if($latestMod.Version -ge [vsteam_lib.Versions]::ModuleVersion) {
+   Write-Host "There is a new version of VSTeam available ($($latestMod.Version)). Run 'Update-Module -Name VSTeam' to update."
```

Comment:
maybe in color yellow?

Reply:
I am having second thougths on this because it really adds a lot of time to the module import time. Hey, @smurawski is there a better way for a module to check for a newer version?

Reply:
It goes from instant to a very noticeable amount of time.